### PR TITLE
Add the configuration usgae of `postgresql-datasource`.

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

**Comment by liweinan** [comment]
The solution works with:

```bash
$ mvn wildfly:run -P postgres
```

Here is the server output:

```
16:46:57,591 INFO  [org.jboss.as.connector.subsystems.datasources] (MSC service thread 1-5) WFLYJCA0001: Bound data source [java:jboss/datasources/PostgreSQLDS]
```

And I can see the relative SQL log output from PostgreSQL:

```sql
2024-11-07 16:00:05.664 CST [58270] LOG:  execute <unnamed>: SELECT JOB_EXECUTION.JOBEXECUTIONID FROM JOB_EXECUTION INNER JOIN JOB_INSTANCE ON JOB_EXECUTION.JOBINSTANCEID=JOB_INSTANCE.JOBINSTANCEID WHERE JOB_EXECUTION.BATCHSTATUS IN ('STARTED', 'STARTING') AND JOB_INSTANCE.JOBNAME=$1
2024-11-07 16:00:05.664 CST [58270] DETAIL:  parameters: $1 = 'simpleBatchletJob'
```

**Comment by liweinan** [comment]
Now it contains a new profile `postgres-default-ds`, and I split the `h2` database configuration into a default profile.

In addition, I updated the README accordingly.

File: deployment/pom.xml [hunk 1]
```diff
@@ -183,6 +102,90 @@
...
+                                </goals>
+                            </execution>
+                            <execution>
+                                <phase>post-integration-test</phase>
+                                <id>wildfly-stop</id>
+                                <goals>
+                                    <goal>shutdown</goal>
+                                </goals>
+                            </execution>
+                        </executions>
```
Comment (jamezp) [inline deployment/pom.xml:184]: This can be moved out of the profile so it doesn't need to be duplicated.

**Comment by jamezp** [comment]
This could be used as an alternative https://github.com/jberet/jberet-examples/pull/16

**Pull request closed without merging**